docs/python/debugging.md

```diff
@@ -95,7 +117,15 @@ Specifies how program output is displayed.
 
 ### `cwd`
 
-Specifies the current working directory for the program being debugged. The standard configuration uses `${workspaceFolder}` to use the project folder. If not specified, `cwd` defaults to the current folder open in VS Code.
+Specifies the current working directory for the debugger, which is the base folder for any relative paths used in code. If omitted, defaults to the folder of the file specified in `program`. A common setting is to use `${workspaceFolder}` (the folder open in VS Code) with or without additional subfolders.
```

Comment:
Let me double check this logic and get back.

Reply:
Please do...this seems to be the behavior from the tests I ran, but verifying from the code is better. :)

Reply:
1. If you open a workspace in VS Code, then the default is to use the `${workspaceFolder}`. 
2. If you do not open a workspace in VS Code, but attempt to debug an arbitrary python file then the default is the directory of the python file.

Reply:
@brettcannon I'm kinda having a brain freeze right now. Please could you confirm this.

Reply:
You can't debug an arbitrary file as VS Code demands a workspace, so I think you only need to care about option 1 and I think that's right. 😉

Reply:
OK, I ran some more tests and confirmed ${workspaceFolder} as the default when cwd is omitted.